docs/building/windows-instructions.md

```diff
@@ -69,7 +69,7 @@ After successfully building, run tests in the Visual Studio Test Explorer window
 
 ### Running tests from the command line
 
-From the root, run `build.cmd` and then `build.cmd -runTests`.
+From the root, run `build.cmd` and then `build.cmd -test`.
```

Comment:
Also, to run all tests, they now need to also include the "-integrationTest" flag (at the same time as "-test". Else, it doesn't run the Integration tests (aka the Functional tests)